plugins/woocommerce/src/Blocks/Shipping/ShippingController.php

```diff
@@ -129,16 +129,40 @@ public function show_local_pickup_details( $return_value, $order ) {
 		$details         = $shipping_method->get_meta( 'pickup_details' );
 		$location        = $shipping_method->get_meta( 'pickup_location' );
 		$address         = $shipping_method->get_meta( 'pickup_address' );
+		$cost            = $shipping_method->get_total();
 
-		if ( ! $address ) {
+		$section = '';
+
+		if ( $location ) {
+			$section .= sprintf(
+				// Translators: %s location name.
+				__( 'Collection from <strong>%s</strong>:', 'woocommerce' ),
+				$location
+			);
+		}
+
+		if ( $address ) {
+			$section .= '<br/><address>' . nl2br( esc_html( str_replace( ',', ', ', $address ) ) ) . '</address>';
```

Comment:
We end up with an extra line break after this because `<address>` is block level. Do you think it would work better if we wrapped each item with `<p>` or as a list instead of using line breaks?